CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # MicroProfile Language Server Changelog
 
+## [0.12.0](https://github.com/eclipse/lsp4mp/milestone/13?closed=1) (August 14, 2024)
+
+### Enhancements
+
+- Add support for @Startup healthcheck diagnostic. See [#443](https://github.com/eclipse/lsp4mp/issues/443).
+
+### Bug Fixes
+
+- Unrecognized property mp.messaging.* when Channel annotation is used along with Multi. See [#451](https://github.com/eclipse/lsp4mp/pull/451).
+- fix: synchronized projectLabels + collect codeLens when project is loaded. See [#444](https://github.com/eclipse/lsp4mp/pull/444).
+
+### Build
+
+- Update to the latest jdt-ls. See [#446](https://github.com/eclipse/lsp4mp/issues/446).
+- Add About files to JDT extension bundles & Maven artifacts. See [#437](https://github.com/eclipse/lsp4mp/pull/437).
+
+### Other
+
+- Update eclipse.jdt.ls to 1.34.0-SNAPSHOT. See [#441](https://github.com/eclipse/lsp4mp/pull/441).
```

Comment:
This is also a build change. However, we already list the PR where we update to 1.37.0-SNAPSHOT, so I think we should remove this entry from the changelog

Reply:
Right! Removed.